apps/application-system/api/src/app/modules/application/application.controller.ts

```diff
@@ -1007,4 +1008,45 @@ export class ApplicationController {
 
     return { url }
   }
+
+  @Scopes(ApplicationScope.read)
+  @Get('applications/:id/attachments/:s3Key/presigned-url')
+  @ApiParam({
+    name: 'id',
+    type: String,
+    required: true,
+    description: 'The id of the application which the file was created for.',
+    allowEmptyValue: false,
+  })
+  @ApiParam({
+    name: 's3Key',
+    type: String,
+    required: true,
+    description: 's3Key',
+    allowEmptyValue: false,
+  })
+  @ApiOkResponse({ type: PresignedUrlResponseDto })
```

Comment:
Checkout the @Documentation decorator from @island.is/nest/swagger.

Reply:
hi looks good from what i see here, https://github.com/island-is/island.is/blob/main/libs/nest/swagger/README.md, not superclear what i need to import to get it running ...

Reply:
amazingly figured it out
`import { Documentation } from '@island.is/nest/swagger'
`